### ISSUE TITLE
Would like a semver number in flutter_tool usage data

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#13641]
Would be useful to me to have the flutter/flutter semver number included in the flutter_tools usage information.  Right now it's alpha/12345abc, but would be easier to read in the usage tables if it inlucded 0.0.19, vs 0.020, etc.

I believe this is the code in question:
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/version.dart
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/usage.dart

We're already increasing the semver number as part of our release process:
https://github.com/flutter/flutter/wiki/Release-process

Thoughts @tvolkert?

**Comment by tvolkert** [comment]
We'd need a way to extract the semver number from our pubspec.yaml file - relevant code is here:

https://github.com/flutter/flutter/blob/15601fe55cdafe3d854f0125dde1b47b1be2a526/packages/flutter_tools/lib/src/flutter_manifest.dart#L37

However, @Hixie and I are considering nuking the semver number altogether from our pubspec and just git tagging releases once they've baked long enough.  In that design, we could use `git tag --contains <commit>` to see if our current version has a semantic version attached to it.

tldr; let's discuss in the new year.

**Comment by Hixie** [comment]
@eseidel How does this look now?

**Comment by eseidelGoogle** [comment]
This is how it's reported now:

<img width="263" alt="screen shot 2018-05-22 at 2 20 21 pm" src="https://user-images.githubusercontent.com/11857803/40390834-4ef19c88-5dcb-11e8-86a0-85af83e38500.png">
